src/styles/blocks/card.css

```diff
@@ -0,0 +1,96 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  display: flex;
+  object-fit: contain;
+  width: 100%;
+  height: 174px;
+
+  padding: 0 3px 40px;
+  margin: 0;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+
+  padding-bottom: 4px;
+  margin: 0;
+}
+
+.card__code {
+  font-size: 10px;
+  color: #616070;
+  line-height: 14px;
+
+  padding-bottom: 16px;
+  margin: 0;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding-bottom: 24px;
+}
+
+.card__count-reviews {
+  font-size: 10px;
+  line-height: 14px;
+  margin: 0;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding-bottom: 16px;
+}
+
+.card__price-text {
+  color: #616070;
+
+  margin: 0;
+}
+
+.card__price-value {
+  font-weight: bold;
+  font-size: 16px;
+
+  margin: 0;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  font-weight: bold;
+  font-size: 14px;
+  text-transform: uppercase;
+  color: #fff;
+  text-decoration: none;
+
+  height: 40px;
```

Comment:
As for me setting paddings is better than height hardcoding, probalby sizes of this link should be different if we'll want to change font-size